screen/wallets/home.js

```diff
@@ -128,6 +129,16 @@ const WalletHome = ({ navigation }) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [wallets]);
 
+  useEffect(() => {
+    if (!wallets) return;
+    const interval = setInterval(() => {
+      console.log('##### refreshing balances #####');
+      if(isLoading) return;
+      refreshBalances().catch(console.error);
+    }, 20 * 1000);
+    return () => clearInterval(interval);
+  }, []);
```

Comment:
I think `isLoading` should be added to the useEffect dependencies, otherwise it will not have the current value in the interval callback.

Reply:
Thanks, the truth is that no guard is needed at all, I'm removing it.